src/filter/types.ts

```diff
@@ -10,6 +10,12 @@ export type ArrayOperator =
   | {
       $last?: FilterOperator
     }
+  | {
```

Comment:
Is this right?

Reply:
Looks right I just cleaned it up into it's own type